autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/saving/AutofillSavingCredentialsDialogFragment.kt

```diff
@@ -41,9 +44,14 @@ import javax.inject.Inject
 @InjectWith(FragmentScope::class)
 class AutofillSavingCredentialsDialogFragment : BottomSheetDialogFragment(), CredentialSavePickerDialog {
 
+    override fun getTheme(): Int = R.style.AutofillBottomSheetDialogTheme
```

Comment:
I'd hoped so initially, but I didn't find a way within a reasonable period of time. It was incredibly frustrating to find the right combo of setting dialog theme and use of the styles to get the bottom sheets to appear correctly (rounded corners) in all cases because Google have Strong Opinions™ against this. This was the ultimate combo I landed on which met all the requirements.

I'm sure there are better ways to do it, but i'm not wanting to spend any longer looking for improvements here, tbh. 😫 (welcome any suggestions you have off-hand!)